### ISSUE TITLE
dotnet tool install --source-feed isn't relative

### ISSUE THREAD

**Issue by richlander** [dotnet/cli#9132]
## Steps to reproduce

```console
C:\git\PowerShell\src\powershell-win-core>dotnet tool install -g pwsh --version 6.1.0-preview.1 --source-feed .\nupkg
C:\Program Files\dotnet\sdk\2.1.300-preview2-008530\NuGet.targets(114,5): error : The local source 'C:\Users\rlander\AppData\Local\Temp\xrucfdsj.yyf\nupkg' doesn't exist. [C:\Users\rlander\AppData\Local\Temp\xrucfdsj.yyf\restore.csproj]
The tool package could not be restored.
Tool 'pwsh' failed to install. You may need to specify the version using

   dotnet tool install -g pwsh --version <version>

```

## Expected  behavior

I was expecting the NuGet source to realize I wanted it to use: `C:\git\PowerShell\src\powershell-win-core\nupkg`

It uses: `'C:\Users\rlander\AppData\Local\Temp\xrucfdsj.yyf\nupkg`

Fully-qualified path works. I expect to be able to use relative paths.

## Environment data
dotnet --version
2.1.300-preview2-008530

**Comment by livarcocc** [comment]
cc @wli3 @peterhuene please take a look.

**Comment by peterhuene** [comment]
This appears to be a regression caused by https://github.com/dotnet/cli/commit/9cc2b7cd2f9dc165ee3a04b1de6fa5065b142dd3.  That commit moved the source feed options to the project file, making them relative to the temporary directory instead of where the restore is run (i.e. the current directory).

I'm thinking about a proper fix to this.

**Comment by wli3** [comment]
We can go back to pass by command line. But we need to properly escape the path. (that's why i move to xml first). use --source source 1 --source source2 can achieve multiple source.

Or we can expend the relative path ourself.

**Comment by wli3** [comment]
I think we should just GetFullName of the path, and add full path to temp project

**Comment by peterhuene** [comment]
Sources aren't just paths.

**Comment by wli3** [comment]
you are right...

**Comment by peterhuene** [comment]
So getting the full path provided the source is not an absolute Uri should do the trick.

**Comment by peterhuene** [comment]
Fixed with #9137 (note, this is now the `--add-source` option.
